x-pack/plugins/security/server/routes/session_management/index.ts

```diff
@@ -13,5 +13,8 @@ import { defineInvalidateSessionsRoutes } from './invalidate';
 export function defineSessionManagementRoutes(params: RouteDefinitionParams) {
   defineSessionInfoRoutes(params);
   defineSessionExtendRoutes(params);
-  defineInvalidateSessionsRoutes(params);
+
+  if (params.buildFlavor !== 'serverless') {
```

Comment:
I'm actually not sure what the typical use case of the API is. I would guess that we don't need it in serverless because all sessions would come from cloud authentication and should be valid from project to project?

Reply:
If I remember correctly, the main motivation behind that API was to address "emergency" situations when the session index could grow quickly (for instance, with disabled session timeouts and anonymous access). However, I've never heard of anyone actually using it. In the context of Serverless, it's not possible to disable session times and/or enable anonymous access.